cli/src/main/kotlin/com/malinskiy/marathon/cli/config/deserialize/AnalyticsConfigurationDeserializer.kt

```diff
@@ -19,6 +19,13 @@ class AnalyticsConfigurationDeserializer : StdDeserializer<AnalyticsConfiguratio
             if (influxDbConfiguration != null) return influxDbConfiguration
         }
 
+        val graphiteNode = node?.get("graphite")
```

Comment:
It's possible to specify both configurations at the same time. We should produce an error that it's incorrect to provide both influx and graphite at the same time. Or at the very least, produce a warning that we influx configuration will override the graphite one.

Reply:
Done. Added throwing a `ConfigurationException` when both InfluxDB and Graphite configurations are specified.